scripts/Deploy.s.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script, stdJson} from "forge-std/Script.sol";
+import {L1ERC20Bridge} from "src/L1ERC20Bridge.sol";
+import {L2ERC20} from "src/L2ERC20.sol";
+import {Constants} from "test/Constants.sol";
+
+/// @notice Deploy L1 bridge and corresponding token to be minted on L2
+contract Deploy is Script, Constants {
+  using stdJson for string;
+
+  function run() public {
+    // Get the contract address for the L1 ERC20Votes token
+    string memory file = "broadcast/DeployFakeERC20.s.sol/43113/run-latest.json";
```

Comment:
Are these scripts meant to be for the demo only, or would these be the production deploy scripts even for setting this up with a real system? If the latter, it might be cleaner to read the L1 token data from a config file rather than from the broadcast.

Reply:
That is a great question I mainly wrote than out of necessity to test the cross chain functionality. We can still read of the broadcasts and make the file path a variable since the chain id is in the file path. or just make the chain id the config variable and interpolate it in the path.

Generalizing these scripts may be good to handle in separate pr once the entire system has been merged into main.

Reply:
Ok cool, yes I think it would be reasonable to track the generalization of these scripts in a separate issue that we tackle once all the core functionality is reviewed/merged. Thanks!

Reply:
Made an issue here #16